src/components/charts/LLinearChartExpand.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <div
+    class="LLinearChartExpand d-flex flex-column flex-grow-1 pa-4"
+  >
+    <div class="LLinearChartExpand__header d-flex justify-space-between pb-3">
+      <span
+        class="LLinearChartExpand__header__action"
+        @click="expandList"
+      >
+        Recolher
+      </span>
+      <!-- <div>
+        <v-text-field
+          v-model="input"
+          outlined
+          hide-details
+          height="20px"
+          min-height="20px"
+          placeholder="Pesquisar"
+          @keydown.enter="search"
+        />
+      </div> -->
+    </div>
+    <v-simple-table
+      ref="LLinearChartExpand__table"
+      dense
+      height="152px"
+      class="LLinearChartExpand__table"
+    >
+      <template
+        v-slot:default
+      >
+        <thead>
+          <tr>
+            <th
+              v-for="(item, index) in headers"
+              :key="index"
+              class="LLinearChartExpand__table__title text-left"
+            >
+              {{ item }}:
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr
+            v-for="(item, index) in data"
+            :key="index"
+            class="LLinearChartExpand__table__line"
+          >
+            <td>{{ index + 5 }}</td>
+            <td>{{ item.label }}</td>
+            <td class="LLinearChartExpand__table__line__value">
+              {{ item.value }} USD
+            </td>
+            <td>{{ item.total }}</td>
+          </tr>
+        </tbody>
+      </template>
+    </v-simple-table>
+    <div
+      v-if="loading"
+      class="d-flex justify-center LLinearChartExpand__loading pt-2"
+    >
+      <l-loading
+        size="20px"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import LLoading from '~/src/components/loadings/LLoading'
+
+export default {
+  components: {
+    LLoading
+  },
+  props: {
+    data: {
+      type: Array,
+      default: () => ([])
+    },
+    headers: {
+      type: Array,
+      default: () => ([])
+    },
+    loading: Boolean
+  },
+  data () {
+    return {
+      input: null,
+      bottom: false,
+    }
+  },
+  watch: {
+    'bottom' (bottom) {
```

Comment:
```suggestion
    bottom (bottom) {
```
dessa maneira, deveria funcionar também?